scripts/integration.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -ex
+set -o pipefail
+
+if [ ! -f genfiles/server.key ]; then
+	./scripts/prepare_server.sh -f
+fi
+
+docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
+trap "docker-compose down" INT EXIT
+
+set +e
+wget -t 60 --spider --retry-connrefused --waitretry=1  0.0.0.0:443 || sleep 2
```

Comment:
nit: 0.0.0.0 isn't typically used as an address to connect to. For a server, it means listen on all interfaces but it doesn't have obvious meaning for a client (Wikipedia says "In the Internet Protocol Version 4, the address 0.0.0.0 is a non-routable meta-address used to designate an invalid, unknown or non-applicable target"). It works for wget because it special-cases 0.0.0.0 to be equivalent to 127.0.0.1, but not all clients do this, so it's a bit confusing to encounter. The clearest thing to do is use 127.0.0.1.